Build/Customizations/Projects.props

```diff
@@ -1,15 +1,35 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<NormalTestConfiguration Condition="'$(NormalTestConfiguration)' == ''">
+			Win_NET48 + NET48 + NoBrowser + NoDB + Debug + x86;
+			Win_NET48 + NET48 + NoBrowser + NoDB + Release + x86;
+			Win_NET48 + NET48 + NoBrowser + NoDB + Debug + x64;
+			Win_NET48 + NET48 + NoBrowser + NoDB + Release + x64;
+
+			<!-- Local-->
+			LocalMachine + NET48 + NoBrowser + NoDB + Debug + x86;
+			LocalMachine + NET48 + NoBrowser + NoDB + Release + x86;
+			LocalMachine + NET48 + NoBrowser + NoDB + Debug + x64;
+			LocalMachine + NET48 + NoBrowser + NoDB + Release + x64;
+
+			<!-- Exercise compatibility between installed .NET version, target framework and SQL Server -->
+			Win_NET48 + NET472 + NoBrowser + NoDB + Release + x64;
+			Win_NET472 + NET472 + NoBrowser + NoDB + Release + x64;
```

Comment:
MK told us yesterday that we need to test NET462 as well, so we'd need a test configuration with NET462 here was my understanding